pkgs/applications/misc/sweethome3d/default.nix

```diff
@@ -53,6 +53,9 @@ let
     nativeBuildInputs = [ makeWrapper unzip autoPatchelfHook ];
     buildInputs = [ ant jdk p7zip gtk3 gsettings-desktop-schemas libXxf86vm ];
 
+    # upstream targets Java 7 by default
+    env.ANT_ARGS = "-DappletClassSource=8 -DappletClassTarget=8 -DclassSource=8 -DclassTarget=8";
```

Comment:
would there be any benefits to targeting a more recent version of Java other than 8? Since the build message suggests _8 or later_

Reply:
Targeting a later version will require actual patching to the buildfile, because it uses a feature that was removed after java8. It could be done, but I just wanted to fix the build.